app/main.py

```diff
@@ -1,34 +1,76 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple[int, int],
+            end: tuple[int, int],
+            is_drowned: bool = False
+    ) -> None:
+        self.start = start
+        self.end = end
+        self.is_drowned = is_drowned
+        self.decks = []
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+        if self.start == self.end:
+            self.decks.append(Deck(start[0], start[1]))
+        if self.end[1] > self.start[1]:
+            for cor_y in range(self.start[1], self.end[1] + 1):
+                self.decks.append(Deck(self.start[0], cor_y))
+        if self.end[0] > self.start[0]:
+            for cor_x in range(self.start[0], self.end[0] + 1):
+                self.decks.append(Deck(cor_x, self.start[1]))
```

Comment:
It's better to move to separate method like `fill_decks` and add some comments or better naming, for clear understanding what are you doing here